crates/binjs_io/src/lib.rs

```diff
@@ -254,9 +255,9 @@ impl Format {
             Format::Multipart { stats, .. } =>
```

Comment:
Note: You'll need to rebase this change, as we removed `Format::Multipart`, but that shouldn't be too difficult.

Reply:
I don't get this. I could see `Format::Multipart` in the master.

Reply:
Sorry, misread, I thought it was `Multistream`.

My bad :)